### PR TITLE
chore: dedupe and update packages

### PR DESCRIPTION
**Pull request by radium-v**
# Description

- Dedupe `lodash-es`, `@babel/*`, and `typescript`
- Update `docusaurus` and `storybook` addons
- Lock specific versions for `api-extractor@7.8.1`, `lodash-es@4.7.15`
- Regenerate lockfile
- Don't run `build` when running `test`

To test it locally, make sure to do a full clean with `yarn lerna clean` and `git clean` before running `yarn install`.

## Motivation & context
Cuts dependencies down from >1GB to ~634MB.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: examples/todo-app-tutorial/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@microsoft/fast-colors": "^5.0.12",
     "@microsoft/fast-element": "latest",
     "@microsoft/fast-foundation": "latest",
-    "lodash-es": "^4.17.15",
+    "lodash-es": "4.17.15",
```
Comment (awentzel) [inline examples/todo-app-tutorial/package.json:18]: @radium-v why are we pinning this version?

Comment (radium-v) [reply examples/todo-app-tutorial/package.json:18]: It's due to this bug (lodash/lodash#5033) in `4.17.20`. Pretty much everything breaks with that latest version.

An alternative to pinning could be setting a top-level resolution entry, but I don't know how that will affect nested dependencies.

File: packages/tooling/fast-figma-plugin-msft/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "watch": "webpack --progress --watch --mode=production",
     "prettier": "prettier --config ../../../.prettierrc --write \"**/*.{ts,tsx}\"",
     "prettier:diff": "prettier --config ../../../.prettierrc \"**/*.{ts,tsx}\" --list-different",
-    "test": "yarn eslint && yarn build",
+    "test": "yarn eslint",
```
Comment (nicholasrice) [inline packages/tooling/fast-figma-plugin-msft/package.json:12]: What's the rational for *not* building as part of the test pass?

Comment (radium-v) [reply packages/tooling/fast-figma-plugin-msft/package.json:12]: Building after testing is redundant in more situations than not. I'll move these specific changes to a different PR with more details.

Comment (radium-v) [reply packages/tooling/fast-figma-plugin-msft/package.json:12]: #4336

File: packages/utilities/fast-web-utilities/package.json [hunk 3]
```diff
@@ -70,24 +70,21 @@
...
     "eslint-loader": "^4.0.0",
     "jest": "^25.4.0",
-    "lodash-es": "^4.17.10",
+    "lodash-es": "4.17.15",
     "prettier": "2.0.2",
     "ts-jest": "^25.4.0",
     "ts-loader": "^4.0.1",
     "typescript": "^3.9.0"
   },
-  "peerDependencies": {
```
Comment (nicholasrice) [inline packages/utilities/fast-web-utilities/package.json:88]: Can you explain the rational for removing the peer dependency? Won't this cause "module not found" error if the author uses this package but doesn't explicitly install `lodash-es`? I understand they need to do that either way, but w/ peerDependencies they get a warning that they need to install `lodash-es`

File: packages/utilities/fast-animation/package.json [hunk 4]
```diff
@@ -105,8 +104,5 @@
     "webpack-bundle-analyzer": "^3.5.2",
     "webpack-cli": "^3.2.1",
     "webpack-dev-server": "^3.11.0"
-  },
-  "peerDependencies": {
```
Comment (nicholasrice) [inline packages/utilities/fast-animation/package.json:109]: Same question here

File: packages/utilities/fast-web-utilities/package.json (hunk 3 above)
Comment (chrisdholt) [reply packages/utilities/fast-web-utilities/package.json:88]: This should simply act as a notification that the dependency is needed. If we remove this I don't think we get any benefit in our build, but it does make things more difficult in others' builds.

**Review by nicholasrice — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]
I'm super excited for this!

**Review by chrisdholt — APPROVED** [review]

**Pull request merged**